xbmc/platform/darwin/ios/IOSExternalTouchController.mm

```diff
@@ -103,6 +103,11 @@ - (id)init
   return self;
 }
 //--------------------------------------------------------------
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations
+{
+    return UIInterfaceOrientationMaskLandscape;
```

Comment:
indentation is wrong, correct are 2 spaces

Reply:
right, my bad. thanks!